### DISCUSSION TITLE
ZuluScsi RP2040 with Video Toaster Flyer - Just a little more write speed needed

### DISCUSSION THREAD

**Discussion by JaiYenJohn** [ZuluSCSI/ZuluSCSI-firmware#317] [Q&A] [answered]
Trying to use 3 ZuluScsi RP2040's on an Amiga with the Video Toaster Flyer. One of the first video non-linear editing systems and I'm so very close to getting the minimum sustained write speed needed for it to work, I was hoping someone could recommend a tweek to my setup to help.
![IMG_0294](https://github.com/ZuluSCSI/ZuluSCSI-firmware/assets/148535391/04fcca35-ac05-4080-b7f2-fe2beff1bd34)

According to the FLYER HD TOOLS application a SCSI drive needs a minimum write speed of 4.8 MB/s, and less than 200ms Max Idle time for the highest HQ5 quality. My standard RP2040 using Sandisk Extreme Pro Class 10 SDXC cards is giving me 4.66 MB/s and 280ms Max Idle time for write speeds. Back in the mid-90's the scsi drives at the time weren't fast enough across the whole platter to do HQ5, so this Flyer Tools software was used to analyze your hard drive then "mask off" the slower inner portions of the platter for recording. So you ended up with a smaller capacity partition, but one that was fast enough. This was called "short stroking". It also overwrote the Cache built into the drive itself to optimize for sustained read/writes, instead of standard PC tasks. It was really pushing the limits of mid-90's tech. But I assume that short stroking an SD card would be useless since there is no "inner" part spinning slower than the "outer".
![IMG_0296](https://github.com/ZuluSCSI/ZuluSCSI-firmware/assets/148535391/7844579e-7870-4608-9bab-bf207ca3ecae)

As of now the video capture aborts with dropped frames if I try to record in HQ5 mode. If I record in the lower bit-rate standard mode, everything works perfectly fine, so it seems to be just a matter of SCSI write performance.

Is there any tweak I can make to the ZuluScsi firmware or .ini file to eek out just another .3 MB/s or so in writes, and reduce the max idle time below 200?

Attached is my ZuluLog if it helps. Thanks!
[zululog.txt](https://github.com/ZuluSCSI/ZuluSCSI-firmware/files/13052344/zululog.txt)

**Comment by Robert-MyAmigaShop** [comment]
Hi JaiYenJohn,

I know it's been awhile (I just ran across this during my search), but did you ever figure out a solution to this?

Thanks,
Robert

**Comment by JaiYenJohn** [comment, marked as answer]
Just a followup: I ended up using a Zulu Scsi v6.4 board instead of the RP2040 and it works perfectly for HQ5 on the Video Toaster Flyer. I'd recommend not using the RP2040 for A&B video drives, but it still works quite well for the audio C drive.
